docs/subnets/create-a-fuji-subnet.md

```diff
@@ -689,11 +672,14 @@ take effect.
 
 :::warning
 
-If the `join` command isn't successfully completed before `addValidator` is completed and the 
-validator's stake weight is >20% of the Subnet total, the Subnet may have down time.
+If the `join` command isn't successfully completed before `addValidator` is completed, the Subnet 
+may experience degraded performance.
 
 :::
 
+Now that the node has joined your Subnet, if you are a holder of the Subnet control key, you
```

Comment:
Now that the node has joined the Subnet, a Subnet control key holder must call `addValidator`